src/main/java/org/opentripplanner/apis/gtfs/datafetchers/LegImpl.java

```diff
@@ -324,4 +328,16 @@ public DataFetcher<Iterable<Leg>> nextLegs() {
   public DataFetcher<Double> accessibilityScore() {
     return environment -> NumberMapper.toDouble(getSource(environment).accessibilityScore());
   }
+
+  @Override
+  public DataFetcher<Relay.ResolvedGlobalId> id() {
+    return environment -> {
+      var ref = getSource(environment).getLegReference();
+      if (ref == null) {
+        return null;
+      }
+      var id = LegReferenceSerializer.encode(ref);
+      return new Relay.ResolvedGlobalId("Leg", id);
+    };
```

Comment:
I don't think we should use the global id anymore, since we have own query for this now, we can just return a string which is the output of the `LegReferenceSerializer.encode(ref)`.

Reply:
Using plain string is indeed simpler. Changed.